core/storage/exploration/gae_models.py

```diff
@@ -377,6 +380,20 @@ def get_featured(cls):
         ).fetch(feconf.DEFAULT_QUERY_LIMIT)
 
     @classmethod
+    def get_top_rated(cls):
+        """Returns an iterable with the top rated exp summaries that are
+        public in descending order.
+        """
+        return ExpSummaryModel.query().filter(
+            ndb.OR(ExpSummaryModel.status == feconf.ACTIVITY_STATUS_PUBLIC,
+                   ExpSummaryModel.status == feconf.ACTIVITY_STATUS_PUBLICIZED)
+        ).filter(
+            ExpSummaryModel.deleted == False  # pylint: disable=singleton-comparison
+        ).order(
+            -ExpSummaryModel.scaled_average_rating
+        ).fetch(feconf.DEFAULT_QUERY_LIMIT)
```

Comment:
Actually put a separate constant in feconf (near RECENTLY_PUBLISHED_QUERY_LIMIT) and call it 8. I think the default query limit is greater.